src/openrct2/EditorObjectSelectionSession.cpp

```diff
@@ -47,7 +47,7 @@ static void ReplaceSelectedWaterPalette(const ObjectRepositoryItem* item);
 
 /**
  * Master objects are objects that are not
- * optional / required dependants of an
+ * optional / required dependents of an
```

Comment:
This was not a typo. We're using British spelling in the codebase.

> Though dependant is older, dependent dominates in American English; thus, the -ant spelling might be read as a mistake. Additionally, in British English, the spellings function more as indicators of parts of speech than as variants: [dependent](https://www.merriam-webster.com/dictionary/dependent#h1) tends to be used for the adjective and [dependant](https://www.merriam-webster.com/dictionary/dependent#h2) for the noun.

The line break seems odd, though.

```suggestion
 * Master objects are objects that are not optional / required dependants of an object.
```